app/controllers/v0/notifications_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module V0
+  class NotificationsController < ApplicationController
+    include Accountable
+    include ::Notifications::Validateable
+
+    before_action -> { validate_subject!(subject) }
+    before_action :set_account
+
+    def create
+      notification = @account.notifications.build(subject: subject, read_at: read_at)
+
+      if notification.save
+        render json: notification, serializer: NotificationSerializer
```

Comment:
i think you can just do `render json: notification` here and it will automatically use the `NotificationSerializer`

Reply:
Yeah, with as many serializers as we have, I favor being explicit.  Makes it just a little bit easier for the next dev :)